p384/src/arithmetic/scalar/p384_scalar.rs

```diff
@@ -0,0 +1,3813 @@
+//! Autogenerated: './word_by_word_montgomery' --lang Rust --inline p384_scalar
+//! 64 39402006196394479212279040100143613805079739270465446667946905279627659399113263569398956308152294913554433653942643
+//! curve description: p384_scalar
```

Comment:
Where did you get this? Did you generate it yourself?

I don't see pregenerated versions in the fiat-rust tree: https://github.com/mit-plv/fiat-crypto/tree/master/fiat-rust/src

Anyway, this is great!

Reply:
Yes I generated it. It's deterministically reproducible using the command above.

We're using Fiat for p256 scalars in Zig, Go is looking at doing the same thing, so maybe the scalar fields can be added to what gets pregenerated with fiat builds.
But even though some optimizations have been, and keep being made, Fiat CI tasks require a significant amount of time and memory to complete, and I'm not sure that adding these new fields wouldn't hit the CI limits. I'm going to give it a shot.